gammapy/catalog/hawc.py

```diff
@@ -0,0 +1,134 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+HAWC 2FHL catalog
+
+TODO: Add reference_id to the paper?
+"""
+
+from .core import SourceCatalog, SourceCatalogObject
+from ..utils.scripts import make_path
+from astropy.table import Table
+import numpy as np
+
+class SourceCatalogObject2HWC(SourceCatalogObject):
+    """One source from the HAWC 2FHL catalog.
+
+    Catalog is represented by `~gammapy.catalog.SourceCatalog2HWC`.
+    """
+
+    _source_name_key = 'source_name'
+    _source_index_key = 'catalog_row_index'
+
+    def info(self, info='all'):
+        """Print info.
+
+        Parameters
+        ----------
+        info : {'all', 'basic', 'position', ' spectra'}
+            Comma separated list of options
+        """
+        ss = self.__str__(info=info)
+        print(ss)
+
+    def __str__(self, info='all'):
```

Comment:
There is no way to pass parameters to `__str__`. So it doesn't make sense to have an `info='all'` argument here.
Look at the other catalogs how it's done, and if the same mistake is there as well, fix it or ask how to fix it if unclear.

Reply:
I oriented myself with these lines here:

https://github.com/gammapy/gammapy/blob/master/gammapy/catalog/fermi.py#L80

it is with arguments in 

https://github.com/gammapy/gammapy/blob/master/gammapy/catalog/gammacat.py#L68
https://github.com/gammapy/gammapy/blob/master/gammapy/catalog/hess.py#L116

as well. 
Shall I fix it everywhere? Within this PR or in an additional one?

Reply:
I'll fix the `__str__` to how I prefer it in a separate PR now. Coming in 15 min ..

Reply:
See #1109 for how to write the `__str__` and `info` methods.
(merged in master just now; but no need to rebase this PR)